ios/Exponent/Kernel/Views/EXAppViewController.m

```diff
@@ -41,6 +41,13 @@
 #import <ABI45_0_0React/ABI45_0_0RCTAppearance.h>
 #endif
 
+#if defined(EX_DETACHED)
```

Comment:
I think we no longer need to handle that case (this was necessary for classic standalone builds), but let's leave it as is for now. There are a lot of ExpoKit-specific things to wipe out 😅